js/components/graph/Graph.js

```diff
@@ -791,7 +817,7 @@ export class Graph extends React.Component {
             var currentEdge = svg.edges.current[edge]
             var sourceNode = refLookUp(currentEdge.props.source, svg)
             if (!sourceNode.isSelected()) {
-              currentEdge.setState({ status: "missing" })
+              currentEdge.updateStatus("missing")
```

Comment:
Because this function is now in `Graph`, we shouldn't need to call `currentEdge.updateStatus` with a specific argument. Instead, you can call the `updateEdgeStatus` function directly here.